src/client/common/application/workspace.ts

```diff
@@ -69,7 +69,8 @@ export class WorkspaceService implements IWorkspaceService {
         maxResults?: number,
         token?: CancellationToken
     ): Thenable<Uri[]> {
-        return workspace.findFiles(include, exclude, maxResults, token);
+        const excludePattern = exclude === undefined ? this.searchExcludes : exclude;
```

Comment:
Even if `exclude` is not defined, `this.searchExcludes` should be used. Something like,
```suggestion
        const excludePattern = exclude === undefined ? this.searchExcludes : `{exclude,${this.searchExcludes}}`;
```

Reply:
That's fine for string `GlobPattern`s but won't work if `exclude` is a `RelativePattern`. `RelativePattern`s can be constructed by passing a `WorkspaceFolder` or `string` as the base, but if I wanted to do something like 
```ts
if (typeof exclude === typeof RelativePattern) {
    excludePattern = `{[exclude.base, exclude.pattern].join('/'),${this.searchExcludes}}`;
}
```
`exclude.base` has type string, not URI, so this wouldn't work for files that are remote for example. Is handling `search.excludes` where `exclude` is not undefined really necessary? It's certainly not for my purposes (since I don't have other things to exclude), and I'm the only consumer of this function today.

Reply:
I see, yes this can be skipped for now.